proposals/collection-literals.md

```diff
@@ -112,23 +112,48 @@ Collection literals are [target-typed](https://github.com/dotnet/csharplang/blob
 
     * The behavior of collection literals with collections that are not well-behaved is undefined.
 
-## Constructible collection types (target types)
-[constructible-collection-types]: #constructible-collection-types
+## Conversions
+A collection literal expression is classified as a value.
 
-The following are _constructible collection types_ that can be used to construct collection literals.
-These are the valid _target types_ for collection literals.
+A _collection literal conversion_ permits a collection literal expression to be implicitly converted to a type.
 
-The list is in priority order. If a collection type belongs in multiple categories, the first is used.
+The following collection literal conversions exist from a collection literal:
 
-Actual translation of the literal to the corresponding type is defined [below](#collection-literal-translation).
+* To a single dimensional *array type* `T[]`, or a *span type* `System.Span<T>` or `System.ReadOnlySpan<T>`, where for each element `Ei`:
+  * If `Ei` is an *expression_element* there is an implicit conversion from `Ei` to `T`.
+  * If `Ei` is a *dictionary_element* `Ki:Vi`, then `T` is a _type_ `System.Collections.Generic.KeyValuePair<K, V>` and there is an implicit conversion from `Ki` to `K` and from `Vi` to `V`.
+  * If `Ei` is a *spread_element* with enumerated element type `Si` there is an implicit conversion from `Si` to `T`.
```

Comment:
nice and clear.  do we need a link/description of what "enumerated element type" means?

Reply:
I realized we've defined *iteration type* earlier in the spec. I've switched the references to that term.